package.json

```diff
@@ -13,31 +13,24 @@
   },
   "dependencies": {
     "@nuxtjs/axios": "^5.3.6",
-    "@nuxtjs/proxy": "^1.3.1",
     "dotenv": "^6.2.0",
     "element-ui": "^2.5.4",
-    "mailgun-js": "^0.22.0",
+    "encoding": "^0.1.12",
```

Comment:
I can't see where this package ("encoding") is used. Is it a dependency from another package?

Reply:
Yeah, it's because of [this issue with `node-fetch`](https://github.com/netlify/netlify-lambda/issues/142#issuecomment-483880089)